### PR TITLE
Update sketch.js

### PR DESCRIPTION
**Pull request by jobintweets**
Removed unwanted spaces and keywords

### Community Contribution

**Link to live project, video, or image:**

### Sharing

- [ ] I would be happy to have my project shared on The Coding Train's social media!

**Instagram:**

**Twitter:**

### CODE REVIEW

File: CodingChallenges/CC_003_Snake_game/P5/sketch.js [hunk 1]
```diff
@@ -29,23 +27,16 @@ function mousePressed() {
 
 function draw() {
   background(51);
-
-  if (s.eat(food)) {
+ if (s.eat(food)) {
```
Comment (violetcraze) [inline CodingChallenges/CC_003_Snake_game/P5/sketch.js]: Might be wrong but it looks like a space on this line got deleted. Was that intentional?

Comment (jobintweets) [reply CodingChallenges/CC_003_Snake_game/P5/sketch.js]: Yes. Those were just blank spaces

Comment (violetcraze) [reply CodingChallenges/CC_003_Snake_game/P5/sketch.js]: Sorry I meant the that there was a space missing in the indentation. Added a commit to fix it. Sorry I probably didn't use the best wording.

**Comment by violetcraze** [comment]
Thanks for working on this!

**Pull request merged**